File: src/adapter/festa/fantasia/Pessoa.java
package adapter.festa.fantasia;

// Classe adaptada

public class Pessoa {
	
	private String nome;
	private String bebidaFavorita;
	
	public Pessoa(String nome, String bebidaFavorita) {
		super();
		this.nome = nome;
		this.bebidaFavorita = bebidaFavorita;
	}
	
	public String obterNome() {
		return nome;
	}
	
	public String obterBebidaFavorita() {
		return bebidaFavorita;
	}
	

}

File: src/adapter/festa/fantasia/Pessoa.kt
package adapter.festa.fantasia

// Classe adaptada

class Pessoa(private val nome: String, private val bebidaFavorita: String) {

    fun obterNome(): String {
        return nome
    }

    fun obterBebidaFavorita(): String {
        return bebidaFavorita
    }


}
